### PR TITLE
Volunteers system integration

### PR DESCRIPTION
**Pull request by roy-zahor**
set env variable for volunteers base url
VOLUNTEERS_BASE_URL

### CODE REVIEW

**Comment by roy-zahor** [comment]
@noam-almog @may-ben-arie Please take a look

File: routes/volunteering_routes.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+});
+var passportLib = require('../libs/passport');
+var userRole = require('../libs/user_role');
+var User = require('../models/user').User;
+
+router.get('/', userRole.isLoggedIn(), function (req, res) {
+    User.forge({user_id: req.user.attributes.user_id}).fetch().then(user => {
+        if (user) {
+            var token = passportLib.generateJwtToken(user.attributes.email);
+            res.redirect("http://URL.OF.VOLUNTEERING.SYSTEM.LANDING.PAGE?auth_token=" + token);
```
Comment (noam-almog) [inline routes/volunteering_routes.js]: @roy-zahor  why are we back to the solution where we send the token as a parameter ?
i thought we agreed on transferring it as a cookie

Comment (roy-zahor) [reply routes/volunteering_routes.js]: @noam-almog You are correct, I didn't describe this PR correctly. What I was trying to show is how relatively easy is to use existing libraries to implement the SSO. I didn't mean to use token as a parameter.

Comment (noam-almog) [reply routes/volunteering_routes.js]: @roy-zahor my file changes are almost done i'll keep you guys informed

Comment (roy-zahor) [reply routes/volunteering_routes.js]: @noam-almog Cool! Closing this issue..

**Pull request merged**